contracts/OracleManagerApp.sol

```diff
@@ -1,61 +1,32 @@
 pragma solidity >=0.4.24;
 
 import "@aragon/os/contracts/apps/AragonApp.sol";
-import 'tidbit/contracts/DataFeedOracles/DataFeedOracleBase.sol';
+import 'tidbit/contracts/DataFeedOracles/MedianDataFeedOracle.sol';
 import "./ITidbitDataFeedOracle.sol";
 
-contract OracleManagerApp is AragonApp {
+contract OracleManagerApp is AragonApp, MedianDataFeedOracle {
 
   event DataFeedAdded(address dataFeedAddress);
   event DataFeedRemoved(address dataFeedAddress);
 
   bytes32 public constant MANAGE_DATA_FEEDS = keccak256("MANAGE_DATA_FEEDS");
 
-  ITidbitDataFeedOracle public medianDataFeed;  // tidbit MedianDataFeedOracle to record median data throughout time
-  uint public approvedDataFeedsLength;   // number of approvedDataFeeds
-  mapping(address => bool) public approvedDataFeeds; // dataFeeds approved to be medianized
-  mapping(address => bool) public dataFeedAlreadyRecorded; // transitory data structure useful only during function call recordDataMedian
-
   /**
   * @dev Initializes OracleManagerApp
-  * @param _medianDataFeed The data feed that medianizes approvedDataFeeds and records result throughout time
   */
-  function initialize(ITidbitDataFeedOracle _medianDataFeed)
-    external
+  function initialize(address[] _dataFeedSources, address  _dataSource)
```

Comment:
`_dataSource` is kind of a phantom variable here bc we overrode the permissions functionality. Thoughts on this @mikec ?

Reply:
@emaG3m if that's the case, probably best to remove it and hardcode a `0x000...001` address or something to make it obvious that it's not being used